roles/zookeeper-docker/tasks/start-zookeeper-container.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: Start ZooKeeper container using specified host data directories
+  docker_container:
+    name: "zookeeper_{{ inventory_hostname_short }}"
+    image: zookeeper
+    state: started
+    restart_policy: always
+    network_mode: host
+    published_ports: "{{ zookeeper_client_port }},{{ zookeeper_leader_connect_port }},{{ zookeeper_leader_election_port }}" 
+    env:
+      ZOO_MY_ID: "{{ zookeeper_id }}"
+    etc_hosts: "{{ zookeeper_etc_hosts }}"
+    volumes:
+      - "{{ zookeeper_conf_dir_volume }}"
+      - "{{ zookeeper_data_dir_volume }}"
+      - "{{ zookeeper_data_log_dir_volume }}"
+  when: zookeeper_data_directories_configuration == "provided"
+
+- name: Start ZooKeeper container using default host data directories
+  docker_container:
+    name: "zookeeper_{{ inventory_hostname_short }}"
```

Comment:
Use my_id for constructing docker name (it would allow for more docker containers on the same host machine).